### PR TITLE
Update functions.py

### PR DESCRIPTION
**Pull request by binitahona**
I have added inverse_cutoff function in the functions.py which is a modified form of Cutoff_powerlaw function. It allows the inverse cutoff (1/xc) to be fitted instead of the cutoff energy xc. While using xc as a parameter to be fitted, for xc = 0, we run into singularity which can be avoided by fitting 1/xc.

### CODE REVIEW

**Comment by henrikef** [comment]
Hi @binitahona , can you please add a short description here about what you've added/changed in this pull request?

**Comment by henrikef** [comment]
Looks good to me. One comment, could you change the name of the function to something more descriptive, like `Inverse_cutoff_powerlaw` [also, we usually capitalize class names]

Other than that, I'm not sure why the automated checks are failing. @binitahona , can you run the tests locally? just do `pytest -vvv --pyargs astromodels` in an empty directory inside your virtual environment. Some temp. files may be created but you can delete them after.

File: astromodels/functions/functions.py [hunk 1]
```diff
@@ -337,6 +337,57 @@ def evaluate(self, x, K, piv, index, xc):
 
         return K * np.exp(log_v)
 
+class inverse_cutoff(Function1D):
```
Comment (grburgess) [inline astromodels/functions/functions.py]: Let's try to keep the naming scheme similar to the other functions. These are classes, so it should be capitalized. Also, the name is not so informative. Perhaps expand upon the name to help users?

**Comment by grburgess** [comment]
@henrikef I'm not sure what is going on with these builds lately. They seem to fail randomly and then work. We will need to learn how all this works as it is probably annoying for @giacomov :)

**Comment by binitahona** [comment]
Thanks @henrikef and grburgess, I have changed the name of the function to "Inverse_cutoff_powerlaw" as suggested to maintain the similar naming scheme as other functions.

**Comment by grburgess** [comment]
Thanks for the addition. Let me know when you are finished pushing changes and then we will commit if the tests pass.

**Comment by binitahona** [comment]
@grburgess, I just checked the changes in my environment using "pytest -vvv --pyargs astromodels" and all tests passed. I am finished with the changes. Thanks.

**Pull request merged**